MS failing since at least 2017-12-23

MS has been failing since 2017-12-23

Based on automated runs it appears that MS has not run successfully in 2 days (2017-12-23).


```
  00:03:37 INFO scrapelib: GET - http://billstatus.ls.state.ms.us/members/house/sykes.xml
00:03:38 INFO pupa: save person Kathy Sykes as person_2eff94be-e870-11e7-902c-0242ac110002.json
00:03:38 INFO pupa: save membership 2eff94be-e870-11e7-902c-0242ac110002 membership in ~{"classification": "lower"} as membership_2eff98ec-e870-11e7-902c-0242ac110002.json
00:03:38 INFO pupa: save membership 2eff94be-e870-11e7-902c-0242ac110002 membership in ~{"classification": "party", "name": "Democratic"} as membership_2eff9b4e-e870-11e7-902c-0242ac110002.json
00:03:38 INFO scrapelib: GET - http://billstatus.ls.state.ms.us/members/house/williams-barnes.xml
00:03:39 INFO pupa: save person Sonya Williams-Barnes as person_2f98bf04-e870-11e7-902c-0242ac110002.json
00:03:39 INFO pupa: save membership 2f98bf04-e870-11e7-902c-0242ac110002 membership in ~{"classification": "lower"} as membership_2f98c33c-e870-11e7-902c-0242ac110002.json
00:03:39 INFO pupa: save membership 2f98bf04-e870-11e7-902c-0242ac110002 membership in ~{"classification": "party", "name": "Democratic"} as membership_2f98c594-e870-11e7-902c-0242ac110002.json
00:03:39 INFO scrapelib: GET - http://billstatus.ls.state.ms.us/members/house/currie.xml
00:03:40 INFO pupa: save person Becky Currie as person_3031e918-e870-11e7-902c-0242ac110002.json
00:03:40 INFO pupa: save membership 3031e918-e870-11e7-902c-0242ac110002 membership in ~{"classification": "lower"} as membership_3031ed28-e870-11e7-902c-0242ac110002.json
00:03:40 INFO pupa: save membership 3031e918-e870-11e7-902c-0242ac110002 membership in ~{"classification": "party", "name": "Republican"} as membership_3031ef80-e870-11e7-902c-0242ac110002.json
00:03:40 INFO scrapelib: GET - http://billstatus.ls.state.ms.us/members/house/hopkins.xml
00:03:41 INFO pupa: save person Steve Hopkins as person_30cbbcd2-e870-11e7-902c-0242ac110002.json
00:03:41 INFO pupa: save membership 30cbbcd2-e870-11e7-902c-0242ac110002 membership in ~{"classification": "lower"} as membership_30cbc0f6-e870-11e7-902c-0242ac110002.json
00:03:41 INFO pupa: save membership 30cbbcd2-e870-11e7-902c-0242ac110002 membership in ~{"classification": "party", "name": "Republican"} as membership_30cbc358-e870-11e7-902c-0242ac110002.json
00:03:41 INFO scrapelib: GET - http://billstatus.ls.state.ms.us/members/house/morgan.xml
00:03:42 INFO pupa: save person Ken Morgan as person_3162d8ce-e870-11e7-902c-0242ac110002.json
00:03:42 INFO pupa: save membership 3162d8ce-e870-11e7-902c-0242ac110002 membership in ~{"classification": "lower"} as membership_3162dce8-e870-11e7-902c-0242ac110002.json
00:03:42 INFO pupa: save membership 3162d8ce-e870-11e7-902c-0242ac110002 membership in ~{"classification": "party", "name": "Republican"} as membership_3162df36-e870-11e7-902c-0242ac110002.json
00:03:42 INFO scrapelib: GET - http://billstatus.ls.state.ms.us/members/house/taylor.xml
00:03:43 INFO pupa: save person Cheikh Taylor as person_31fb84ca-e870-11e7-902c-0242ac110002.json
00:03:43 INFO pupa: save membership 31fb84ca-e870-11e7-902c-0242ac110002 membership in ~{"classification": "lower"} as membership_31fb88d0-e870-11e7-902c-0242ac110002.json
00:03:43 INFO pupa: save membership 31fb84ca-e870-11e7-902c-0242ac110002 membership in ~{"classification": "party", "name": "Democratic"} as membership_31fb8b1e-e870-11e7-902c-0242ac110002.json
00:03:43 INFO scrapelib: GET - http://billstatus.ls.state.ms.us/members/house/barnett.xml
no pupa_settings on path, using defaults
ms (scrape, import)
  bills: {}
  people: {}
  committee: {}
  File "/opt/openstates/venv-pupa/src/pupa/pupa/cli/commands/update.py", line 305, in do_handle
  File "/opt/openstates/venv-pupa//bin/pupa", line 11, in <module>
    load_entry_point('pupa', 'console_scripts', 'pupa')()
  File "/opt/openstates/venv-pupa/src/pupa/pupa/cli/__main__.py", line 67, in main
    subcommands[args.subcommand].handle(args, other)
  File "/opt/openstates/venv-pupa/src/pupa/pupa/cli/commands/update.py", line 260, in handle
    return self.do_handle(args, other, juris)
Traceback (most recent call last):
    report['scrape'] = self.do_scrape(juris, args, scrapers)
  File "/opt/openstates/venv-pupa/src/pupa/pupa/cli/commands/update.py", line 173, in do_scrape
    report[scraper_name] = scraper.do_scrape(**scrape_args)
  File "/opt/openstates/venv-pupa/src/pupa/pupa/scrape/base.py", line 111, in do_scrape
    for obj in self.scrape(**kwargs) or []:
  File "/opt/openstates/openstates/openstates/ms/people.py", line 20, in scrape
    yield from self.scrape_legs('lower')
  File "/opt/openstates/openstates/openstates/ms/people.py", line 39, in scrape_legs
    yield from self.scrape_details(chamber, leg, leg_link, role)
  File "/opt/openstates/openstates/openstates/ms/people.py", line 112, in scrape_details
    " a listed party: {}").format(leg_name)
AssertionError: Remove special-casing for this Republican without a listed party: Shane Barnett
```

Visit http://bobsled.openstates.org for more info.

--- a/openstates/ms/people.py
+++ b/openstates/ms/people.py
@@ -106,7 +106,6 @@ class MSLegislatorScraper(Scraper):
             elif leg_name in ('James W. Mathis',
                               'John Glen Corley',
                               'Missy McGee',
-                              'Shane Barnett',
                               'Kevin Ford'):
                 assert not party, ("Remove special-casing for this Republican without"
                                    " a listed party: {}").format(leg_name)

